workflow/lib/preprocess/preprocess.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 from nd2reader import ND2Reader
 
-from preprocess.file_utils import extract_tile_from_filename
+from shared.file_utils import extract_tile_from_filename
```

Comment:
Makes sense to move to shared as you're scaling this across all aspects of the pipeline, right?

Reply:
I actually didn't mean to move this to `shared` (we only use it in preprocessing) and am moving it back in the next PR. But, I will keep a `shared/file_utils.py` file because we definitely want `get_filename` in the shared utils. 

I don't think the location of this `extract_tile_from_filename` function is important though because I actually want to remove it entirely soon. See #4